__device-tests__/pages/editor-page.js

```diff
@@ -381,6 +382,31 @@ export default class EditorPage {
 		return await this.removeBlockAtPosition( position, this.imageBlockName );
 	}
 
+	// =========================
+	// Gallery Block functions
+	// =========================
+
+	async addNewGalleryBlock() {
+		await this.addNewBlock( this.galleryBlockName );
+	}
+
+	async getGalleryBlockAtPosition( position: number ) {
+		return this.getBlockAtPosition( position, this.galleryBlockName );
+	}
+
+	/*
+	async selectEmptyGalleryBlock( block: wd.PromiseChainWebdriver.Element ) {
+		const accessibilityId = await block.getAttribute( this.accessibilityIdKey );
+		const blockLocator = `//*[@${ this.accessibilityIdXPathAttrib }="${ accessibilityId }"]//XCUIElementTypeButton[@name="Gallery block. Empty"]`;
+		const galleryBlockInnerElement = await this.driver.elementByXPath( blockLocator );
+		await galleryBlockInnerElement.click();
+	}
+	*/
```

Comment:
Wondering if this is a WIP 😅do we want to include this commented code?

Reply:
Indeed it is a WIP :smile: I'll remove it. :+1: